mouse.c

```diff
@@ -237,7 +237,9 @@ luaA_mouse_pushstatus(lua_State *L, int x, int y, uint16_t mask)
     return 1;
 }
 
-/** Get or set the mouse coords.
+/* Get or set the mouse coords.
+ *
+ *@DOC_awful_mouse_coords_EXAMPLE@
```

Comment:
Can't you just replace this whole comment with `documented in lib/awful/mouse/init.lua`?